lib/wda/webdriveragent.js

```diff
@@ -422,7 +423,7 @@ class WebDriverAgent {
    */
   async getInstalledWDBundleId () {
     return this.isRealDevice
-      ? await getIBundleIdsByCFBundleName(this.device, WDA_CF_BUNDLE_NAME)
```

Comment:
so how about moving these methods into corresponding classes and make it polymorphic?

Reply:
yes. I also thought we can get `CFBundleNam` in simulator case (ios-simulator does not have such a method for now tho) as future work